extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -243,7 +243,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
 
     // Fetch Stuff
     protected instrumentFetch(): void {
-        if (!this.supportsFetch() || this._fetchInitialized) {
+        if (!this.supportsFetch() || this._fetchInitialized || !window) {
```

Comment:
window check is already in supportsFetch method, add typeof window check in there instead